### PR TITLE
access control blog

### PR DESCRIPTION
**Pull request by nzoschke**
screen shot 2016-07-29 at 11 10 21 am

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
@nzoschke Clear explanation of the feature, and I like the story element a lot. If you could figure out a way to put a bow on the Bob and Alice situation again in the conclusion I think it would be an even stronger finish.

File: _posts/2016-08-01-access.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+* Create new apps
+
+* Deploy to apps
+
+* Scale apps
+
+* Debug app processes
+
+* View app logs
```
Comment (mwarkentin) [inline _posts/2016-08-01-access.md]: get/set envs?

File: _posts/2016-08-01-access.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+While **Operators** can perform infrastructure actions:
+
+* Apply AMI security updates
```
Comment (mwarkentin) [inline _posts/2016-08-01-access.md]: What does this mean? Rack updates?

File: _posts/2016-08-01-access.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+While **Operators** can perform infrastructure actions:
+
+* Apply AMI security updates 
+
+* SSH to instances
+
+* Scale up to larger instances
+
+* Connect to private databases
```
Comment (mwarkentin) [inline _posts/2016-08-01-access.md]: Does this refer to the proxy command? That's not available to devs?

**Comment by nzoschke** [comment]
@mwarkentin thanks for the feedback! I've pushed some tweaks to answer your questions.

Though it turns out I'm a bit ahead of the curve here, the dev vs ops Rack API stuff hasn't landed yet.

**Comment by nzoschke** [comment]
This is stale. Closing without merge.

**Pull request closed without merging**